public/doc/swagger-2.yaml

```diff
@@ -971,4 +971,4 @@ host: localhost
 schemes:
   - https
   - http
-basePath: ""
+basePath: "/r/insights/platform/service-portal"
```

Comment:
@Fryguy 
Should this include the version number or would that be added to every path?

Reply:
See my later comment...I'm going to deal with versions in a follow up PR.